libs/libc/misc/lib_execinfo.c

```diff
@@ -88,3 +88,38 @@ void backtrace_symbols_fd(FAR void *const *buffer, int size, int fd)
       dprintf(fd, "%pS\n", buffer[i]);
     }
 }
+
+/****************************************************************************
+ * Name: backtrace_format
+ *
+ * Description:
+ *  Format a backtrace into a buffer for dumping.
+ *
+ ****************************************************************************/
+
+FAR const char *backtrace_format(FAR char *buffer, int size,
+                                 FAR void *backtrace[], int depth)
+{
+  FAR const char *format = "%0*p ";
+  int i;
+
+  if (size < 1)
+    {
+      return NULL;
+    }
+
+  buffer[0] = '\0';
+  for (i = 0; i < depth && backtrace[i]; i++)
+    {
+      if (size - i * BACKTRACE_PTR_FMT_WIDTH <= 0)
+        {
+          break;
+        }
+
+      snprintf(buffer + i * BACKTRACE_PTR_FMT_WIDTH,
+               size - i * BACKTRACE_PTR_FMT_WIDTH,
+               format, BACKTRACE_PTR_FMT_WIDTH - 1, backtrace[i]);
```

Comment:
```suggestion
               format, BACKTRACE_PTR_FMT_WIDTH, backtrace[i]);
```

Reply:
but no change

Reply:
Sorry, I have changed it back. I don't understand it. The macro `BACKTRACE_PTR_FMT_WIDTH` includes one space after in the `format` string. I have commented below.